components/brave_new_tab_ui/components/privateTab/index.tsx

```diff
@@ -18,12 +20,23 @@ interface Props {
 }
 
 export default class NewPrivateTab extends React.PureComponent<Props, {}> {
+  get isQwant () {
+    return navigator.language === 'de' || navigator.language === 'fr'
```

Comment:
Nit: This can be a static const defined at the module level, since it does not rely on any props or anything to do with the component